src/NetAnalyzers/Core/Microsoft.NetCore.Analyzers/Runtime/UseRegexMembers.cs

```diff
@@ -0,0 +1,155 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the MIT license.  See License.txt in the project root for license information.
+
+using System.Collections.Immutable;
+using System.Linq;
+using Analyzer.Utilities;
+using Analyzer.Utilities.Extensions;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.Diagnostics;
+using Microsoft.CodeAnalysis.Operations;
+
+namespace Microsoft.NetCore.Analyzers.Runtime
+{
+    using static MicrosoftNetCoreAnalyzersResources;
+
+    /// <summary>
+    /// CA1873: <inheritdoc cref="UseRegexIsMatchMessage"/>
+    /// CA1874: <inheritdoc cref="UseRegexCountMessage"/>
+    /// </summary>
+    [DiagnosticAnalyzer(LanguageNames.CSharp, LanguageNames.VisualBasic)]
+    public sealed class UseRegexMembers : DiagnosticAnalyzer
+    {
+        internal const string RegexIsMatchRuleId = "CA1873";
+        internal const string RegexCountRuleId = "CA1874";
+
+        // Regex.Match(...).Success => Regex.IsMatch(...)
+        internal static readonly DiagnosticDescriptor UseRegexIsMatchRuleId = DiagnosticDescriptorHelper.Create(RegexIsMatchRuleId,
+            CreateLocalizableResourceString(nameof(UseRegexIsMatchTitle)),
+            CreateLocalizableResourceString(nameof(UseRegexIsMatchMessage)),
+            DiagnosticCategory.Performance,
```

Comment:
I'm assuming this is informational, right? Is that what Performance category does?

Reply:
> I'm assuming this is informational, right

Yup, that's set by the next line (IdeSuggestion)